src/Symfony/Component/HttpKernel/EventListener/AbstractSessionListener.php

```diff
@@ -151,7 +151,8 @@ public function onKernelResponse(ResponseEvent $event)
             $request = $event->getRequest();
             $requestSessionCookieId = $request->cookies->get($sessionName);
 
-            if ($requestSessionCookieId && $session->isEmpty()) {
+            $isSessionEmpty = $session->isEmpty();
```

Comment:
Need in 6.0 be replaced with the new logic: https://github.com/symfony/symfony/blob/b2e7fcd4f38c01e13fdd8786689672bf73a97402/src/Symfony/Component/HttpKernel/EventListener/AbstractSessionListener.php#L149